helm-charts/konk-service/values.yaml

```diff
@@ -26,17 +26,15 @@ kind:
     pullPolicy: Always
     # -- Overrides the image tag whose default is the chart appVersion.
     tag: v1.19.0
-  resources: {}
+  resources:
     # We usually recommend not to specify default resources and to leave this as a conscious
     # choice for the user. This also increases chances charts run on environments with little
     # resources, such as Minikube. If you do want to specify resources, uncomment the following
     # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
-    # limits:
-    #   cpu: 100m
-    #   memory: 128Mi
-    # requests:
-    #   cpu: 100m
-    #   memory: 128Mi
+    limits:
+      memory: 128Mi
+    requests:
+      memory: 25Mi
```

Comment:
If we need to override this, like to test on a test-runner that has low resources, the ideal way would be to set `resources: {}` to remove them for the test. Unfortunately, a helm bug prevents this: https://github.com/helm/helm/issues/9136 I don't have a solution for this that allows setting a default other than `{}`.

Reply:
@kd7lxl Is there a change that you're proposing or is this blocking your approval of this PR?

Reply:
It's blocking unless you know (or can research) how we can work around this helm bug. I want to maintain the capability of disabling the resource definitions, like when testing in kind. With the current proposal and bug I don't see a way to do that.